### PR TITLE
Fix link to developer changelog

### PR DESCRIPTION
**Pull request by cachedout**
The previous URL was 404ing. :)

### CODE REVIEW

File: docs/devguide/contributing.asciidoc [hunk 1]
```diff
@@ -147,4 +147,4 @@ In most cases `govendor fetch your/dependency@version +out` will get the job don
 
 To keep up to date with changes to the official Beats for community developers,
 follow the developer changelog
-https://github.com/elastic/beats/blob/master/CHANGELOG-developer.md[here].
+https://raw.githubusercontent.com/elastic/beats/master/CHANGELOG-developer.asciidoc[here].
```
Comment (jsoriano) [inline docs/devguide/contributing.asciidoc]: Could you use https://github.com/elastic/beats/blob/master/CHANGELOG-developer.asciidoc so it is rendered?

Comment (cachedout) [reply docs/devguide/contributing.asciidoc]: Absolutely! Done. :)

Comment (jsoriano) [reply docs/devguide/contributing.asciidoc]: Thanks!

**Comment by jsoriano** [comment]
jenkins, test this please

**Comment by dedemorton** [comment]
Build failure is not related to docs, so I am merging.

**Pull request merged**